### DISCUSSION TITLE
When will 3.0.1 be released?

### DISCUSSION THREAD

**Discussion by rahul1** [facebook/docusaurus#9564] [Q&A]
I need this bugfix: https://github.com/facebook/docusaurus/pull/9531

Currently, our docs site has suffered a regression after upgrading to 3.0.0 because of this bug

**Comment by hrumhurum** [comment]
I am waiting for 3.0.1 as well. 3.0.0 contains omissions that broke existing functionality, they thankfully got fixed, but no release yet.

**Reply by slorber** [reply]
I'll release it soon

But keep in mind that you can use canary releases. 

Just after a release they are usually safe (no breaking change) and only contain the bugfixes you need

**Reply by slorber** [reply]
released: https://github.com/facebook/docusaurus/discussions/9598

**Reply by hrumhurum** [reply]
Thank you!
